spec/lib/mail_handler/mail_handler_spec.rb

```diff
@@ -516,6 +516,26 @@ def expect_header_string(fixture_file, header, header_string)
       attr.should == expected_attributes[index]
     end
   end
+
+    describe :get_attachment_attributes do
+
+    it 'should handle an Outlook attachment with HTML generated from RTF' do
+      mail = get_fixture_mail('outlook-encoding-rtf.email')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.